api/v1/swagger.yaml

```diff
@@ -3094,6 +3094,17 @@ paths:
             Filter by parentAbsolutePath; asterisk (*) wildcards ok.
           required: false
           type: string
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
```

Comment:
@shriramshankar Just checked, we can add different case tags to a subject, like I can have
```
 tags = [
    "mytag",
    "Cali",
    "mytaG",
    "MYTAG"
  ]
```
Is this expected behavior?
Also, if we need to filter case insensitive, there are 2 ways:
1. Filter on API layer instead of query level, like hierarchy endpoint. Disadvantage: get all subjects first and then extra processing to filter.
2. store tags in lowercase in db. This is more efficient way to do, if we agree.

I can add this feature to the new story to filter by excludes as well.